### ISSUE TITLE
kitty@0.70.0.7 hash check failed

### ISSUE THREAD

**Issue by graph** [lukesampson/scoop-extras#1659]
```
Installing 'kitty' (0.70.0.7) [64bit]
Loading kitty from cache
Checking hash of kitty ... ok.
kitty (283,5 KB) [====================================================] 100%
Checking hash of kitty ... ERROR Hash check failed!
App:         extras/kitty
URL:         https://www.9bis.net/kitty/?file=kageant.exe#/kageant.exe
First bytes: 4D 5A 90 00 03 00 00 00
Expected:    afd3681f32332eca7e2a323bfee860cb6e8cac2485385e1a3a171efb345b816d
Actual:      4cbe17fd75ec628d7f0ce3b530b227b24878d9ccd971fa13835028bd9bf4b84d

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=kitty%400.70.0.7%3a+hash+check+failed
```

**Comment by rasa** [comment]
Needs to be fixed manually

**Comment by rasa** [comment]
Fixed in #1772
